docs/quickstart/learning-more-lakefs.md

```diff
@@ -0,0 +1,135 @@
+{: .note .note-info }
+
+   ```bash
+docker run --pull always -p 8000:8000 \
+   -e LAKEFS_BLOCKSTORE_TYPE='azure' \
+   -e LAKEFS_BLOCKSTORE_AZURE_STORAGE_ACCOUNT='YourAzureStorageAccountName' \
+   -e LAKEFS_BLOCKSTORE_AZURE_STORAGE_ACCESS_KEY='YourAzureStorageAccessKey' \
+   treeverse/lakefs run --local-settings
+   ```
+
+  </div>
+  <div markdown="1" id="on-google-gcs">
+
+Note: Make sure the Quickstart Docker Compose from the previous steps isn't also running as you'll get a port conflict.
+{: .note .note-info }
+
+   ```bash
+docker run --pull always -p 8000:8000 \
+   -e LAKEFS_BLOCKSTORE_TYPE='gs' \
+   -e LAKEFS_BLOCKSTORE_GS_CREDENTIALS_JSON='YourGoogleServiceAccountKeyJSON' \
+   treeverse/lakefs run --local-settings
+   ```
+where you will replace ```YourGoogleServiceAccountKeyJSON``` with JSON string that contains your Google service account key.
+
+If you want to use the JSON file that contains your Google service account key instead of JSON string (as in the previous command) then go to the directory where JSON file is stored and run the command with local parameters:
+
+   ```bash
+docker run --pull always -p 8000:8000 \
+   -v $PWD:/myfiles \
+   -e LAKEFS_BLOCKSTORE_TYPE='gs' \
+   -e LAKEFS_BLOCKSTORE_GS_CREDENTIALS_FILE='/myfiles/YourGoogleServiceAccountKey.json' \
+   treeverse/lakefs run --local-settings
+   ```
+This command will mount your present working directory (PWD) within the container and will read the JSON file from your PWD.
+
+  </div>
+  <div markdown="1" id="on-minio">
+
+To use lakeFS with MinIO (or other S3-compatible object storage), use the following example:
+
+Note: Make sure the Quickstart Docker Compose from the previous steps isn't also running as you'll get a port conflict.
+{: .note .note-info }
+
+   ```bash
+docker run --pull always -p 8000:8000 \
+   -e LAKEFS_BLOCKSTORE_TYPE='s3' \
+   -e LAKEFS_BLOCKSTORE_S3_FORCE_PATH_STYLE='true' \
+   -e LAKEFS_BLOCKSTORE_S3_ENDPOINT='http://<minio_endpoint>' \
+   -e LAKEFS_BLOCKSTORE_S3_DISCOVER_BUCKET_REGION='false' \
+   -e LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID='<minio_access_key>' \
+   -e LAKEFS_BLOCKSTORE_S3_CREDENTIALS_SECRET_ACCESS_KEY='<minio_secret_key>' \
+   treeverse/lakefs run --local-settings
+   ```
+
+  </div>
+</div>
+
+## Deploying lakeFS
+
+Ready to do this thing for real? The deployment guides show you how to deploy lakeFS [locally](deploy/onprem.html) (including on [Kubernetes](/deploy/onprem.html#k8s)) or on [AWS](/deploy/aws.html), [Azure](/deploy/azure.html), or [GCP](/deploy/gcp.html). 
+
+Alternatively you might want to have a look at [lakeFS Cloud](https://lakefs.cloud/) which provides a fully-managed, SOC-2 compliant, lakeFS service. 
+
+## lakeFS Samples
+
+The [lakeFS Samples](https://github.com/treeverse/lakeFS-samples) GitHub repository includes some excellent examples including: 
+
+* How to implement cross collection consistency on multiple Delta Tables
```

Comment:
`multi-table transaction` might be a better name for `cross collection consistency` - wdyt? happy to update it in the samples as well if we agree on this terminology.

Reply:
I didn't know what `cross collection consistency` was and just copied it from the repo text :D It's it just multi-table transactions then yes that'd be a more accessible term IMHO.

Reply:
cool. i will update the lakeFS sample now, so it will be consistent with the quickstart - if you could update the docs, that would be great @rmoff

Reply:
Done